modules/op-release-notes-1-4.adoc

```diff
@@ -80,6 +80,30 @@ fsGroup:
   type: MustRunAs
 ----
 
+[id="technology-preview-features-1-4_{context}"]
+== Technology Preview features
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+.Technology Preview tracker
+[cols="1,1,1,1",options="header"]
+|===
+| Feature | OCP 4.5 |OCP 4.6 | OCP 4.7
```

Comment:
@preethit shouldn't the version column refer to OpenShift Pipelines versions?

Reply:
@siamaksade, this table is under the [1.4 RN section](https://docs.openshift.com/container-platform/4.7/cicd/pipelines/op-release-notes.html#op-release-notes-1-4_op-release-notes), not at the top unlike the compatibility matrix, hence the OCP versions. On second thought, I wonder if we are repeating the same things in multiple places - compatibility matrix the note on TP features and the TP table. I will work on consolidating this information and run it by you.

BTW my GH ID is @Preeticp.  preetihit is another person on the OCP team :)

Reply:
@Preeticp Sorry for the mixup :)